### DISCUSSION TITLE
日本で必要な Generic Font Family の議論

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq#392] [General]
2023-12-19 のJLReq TFミーティングを受けて、[Generif Font Family の議論](https://github.com/w3c/i18n-discuss/wiki/Generic-font-families)に下の三つのフィードバックを出しました。

https://github.com/w3c/i18n-discuss/issues/34 kai is also one of popular typefaces in Japan
https://github.com/w3c/i18n-discuss/issues/35 Add "rounded (sans-serif)" as a generic font family
https://github.com/w3c/i18n-discuss/issues/36 Add Kyokasho-tai as a generic font family

これらに関するコメントは直接それらのIssueへお願いします。その他のディスカッションはここで

**Comment by xfq** [comment]
Japanese also has Fangsong fonts (like [イワタ宋朝体](https://www.iwatafont.co.jp/font/sousp.html)), but they don't come with the operating system, and are probably not as popular as other major font styles?

**Reply by himorin** [reply]
Several other Japanese font vendors also provides Fangsong ones, like [DynaFont](https://www.dynacw.co.jp/product/product_download_detail.aspx?sid=3275), and used for graphic design (poster etc.), but I suppose we cannot state these are common among wide range of people - like not included in system fonts. Bunch of font styles are used for commercial designs, like during over 50+ or more years, but most are not for daily (e.g. body text of newspaper or magazine) use I believe...

**Reply by kidayasuo** [reply]
I agree with what @himorin explained.

「楷書」「行書」「草書」「隷書」are regarded as authentic typefaces, and most major font vendors have them in their library.「仿宋 (宋朝 in Japan) 」is less common but some vendors have it.

The modern use of these typefaces is limited to their niches. On digital their use is pretty much limited to designing logos, except kai has a bit wider use cases. You rarely see them in digital documents due to, I think, the fact that their designs are optimized for vertical writing, and also that screen resolution had not been suitable for these typefaces. Is the situation similar or different in China?

If you stroll through Kyoto city, you will find many of these typefaces, including Fangsong, used in the logos of stores, temples, and shrines, which adds to the enjoyment of exploring the city. :)
